packages/apps/spaces/components/ui-kit/organisms/side-panel/SidePanel.tsx

```diff
@@ -22,6 +22,28 @@ export const SidePanel: React.FC<SidePanelProps> = ({
 }) => {
   const router = useRouter();
   const logoutUrl = useRecoilValue(logoutUrlState);
+
```

Comment:
this logic was already moved out to dedicated hook, please use it instead
`useDetectClickOutside`